GroceryStore/AppDelegate.m

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    return YES;
+}
+
+- (void)applicationWillResignActive:(UIApplication *)application {
+    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
+    // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
```

Comment:
Usually we would ask that "boilerplate" stuff that is inserted by Xcode's project or file templates be deleted.  So all of these empty methods that Xcode "helpfully" added for you should be deleted.